libs/define.js

```diff
@@ -0,0 +1,40 @@
+// In order to make our code structured, modular and reusable
```

Comment:
My idea for libs was to put folders in it and source code in folders, not directly in libs.
Let we create folder (lib in libs) called util (or lang, or core, or find proper name) for define.js.